tools/engine_tool/test/build_command_test.dart

```diff
@@ -360,11 +360,11 @@ void main() {
         '//flutter/fml:fml_arc_unittests',
       ]);
       expect(result, equals(0));
-      expect(testEnv.processHistory.length, greaterThanOrEqualTo(2));
-      expect(testEnv.processHistory[3].command[0], contains('ninja'));
-      expect(testEnv.processHistory[3].command[2], endsWith('/host_debug'));
+      expect(testEnv.processHistory.length, greaterThan(6));
```

Comment:
Changed to a strictly greater-than check so that it matches the processHistory index we refer to, which will make it a little more obvious to people who modify this in the future.